src/AppRoutes.tsx

```diff
@@ -81,12 +86,12 @@ export const AppRoutes = () => (
       <Route
         path="administration"
         element={
-          <ProtectedRoute role={AUTH_ROLE.Admin}>
+          <ProtectedRoute title="Administration" role={AUTH_ROLE.Admin}>
             <AdministrationPage />
           </ProtectedRoute>
         }
       />
     </Route>
-    <Route path="*" element={<NotFoundPage />} />
+    <Route path="*" element={<ErrorPage errorDescription="The requested resource could not be found."></ErrorPage>} />
```

Comment:
I would use also `title` attribute `Page not found`, but define it inside https://github.com/project-ncl/pnc-web-ui-react/blob/main/src/utils/PageTitles.ts like `pageNotFound`